code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -229,6 +242,17 @@
 	attack_verb = list("attacks", "hits", "bludgeons")
 
 
+//allows you to replace the bone setter in switchtools with it being a setter child rather than a bonegel child
+/obj/item/weapon/bonesetter/bone_mender
+	name = "bone mender"
+	desc = "A favortie among skeletons, it even sounds like one too."
```

Comment:
hon hon hon

Reply:
Obviously there's also one here, since they say the same thing